vendor/darwin/Foundation/NSWindow.odin

```diff
@@ -589,6 +594,10 @@ View :: struct {using _: Responder}
 View_initWithFrame :: proc "c" (self: ^View, frame: Rect) -> ^View {
 	return msgSend(^View, self, "initWithFrame:", frame)
 }
+@(objc_type=View, objc_name="bounds")
+View_bounds :: proc "c" (self: ^View) -> ^Rect {
+	return msgSend(^Rect, self, "bounds")
```

Comment:
Bounds does not return a pointer to Rect.
https://developer.apple.com/documentation/appkit/nsview/1483817-bounds?language=objc